DancingGoat/Areas/Admin/AppSettingProvider.cs

```diff
@@ -178,5 +181,40 @@ public static string KenticoCloudUrl
                 }
             }
         }
+
+        public static string[] ResponsiveWidths
+        {
+            get
+            {
+                if (_responsiveWidths != null)
+                {
+                    return _responsiveWidths;
+                }
+                else
+                {
+                    var setting = ConfigurationManager.AppSettings[RESPONSIVE_WIDTHS];
+                    _responsiveWidths = string.IsNullOrEmpty(setting) ? new string[] { } : setting.Split(',');
+
+                    return _responsiveWidths;
+                }
+            }
+        }
+
+        public static bool ResponsiveImagesEnabled
```

Comment:
Better to use:
`public static bool ResponsiveImagesEnabled => ResponsiveWidths.Count() > 0;`